go/kbfs/libkbfs/init.go

```diff
@@ -62,6 +62,15 @@ const CtxInitID = "KBFSINIT"
 // AdditionalProtocolCreator creates an additional protocol.
 type AdditionalProtocolCreator func(Context, Config) (rpc.Protocol, error)
 
+const (
+	configModePrefixStr            = "kbfs."
+	configModeStr                  = configModePrefixStr + "mode"
+	configBlockCachePrefixStr      = configModePrefixStr + "block_cache."
+	configBlockCacheMemMaxBytesStr = configBlockCachePrefixStr + "mem_max_bytes"
+	configBlockCacheDiskMaxFracStr = configBlockCachePrefixStr + "disk_max_fraction"
+	configBlockCacheSyncMaxFracStr = configBlockCachePrefixStr + "sync_max_fraction"
```

Comment:
this seems harder to read than (and without significant benefits over) fully specifying these strings:

```golang
configModeStr = "kbfs.mode"
configBlockCacheMemMaxBytesStr = "kbfs.mode.mem_max_bytes"
configBlockCacheDiskMaxFracStr = "kbfs.block_cache.disk_max_fraction"
configBlockCacheSyncMaxFracStr = "kbfs.block_cache.sync_max_fraction"
```

Reply:
Oh oops, I didn't mean to have this commit in this PR, was supposed to be a separate one for KBFS reviewers.  I'll move it out, but also take your advice.  I was trying to minimize copy/paste of course, but happy to have it more readable if you think that's better.